core/src/main/resources/reference.conf

```diff
@@ -85,6 +85,13 @@ system {
 
   # Default number of cache read workers
   number-of-cache-read-workers = 25
+  
+  io {
+    # Global Throttling - This is mostly useful for GCS and can be adjusted to match
+    # the quota availble on the GCS API
+    number-of-requests = 100000
+    per = 100 seconds
```

Comment:
I'd prefer to keep it the way it is here as I can imagine we'll want to support other clouds with different quota denominations in the not too distant future.  😄